src/char/barbarian.py

```diff
@@ -46,7 +47,7 @@ def _do_hork(self, hork_time: int):
         if self._skill_hotkeys["find_item"]:
             keyboard.send(self._skill_hotkeys["find_item"])
             wait(0.5, 0.15)
-        pos_m = self._screen.convert_abs_to_monitor((0, -20))
+        pos_m = Screen().convert_abs_to_monitor((0, -20))
```

Comment:
Maybe make those functions class level too? Since you initializing Screen it at class level now?